pkgs/http/lib/src/browser_client.dart

```diff
@@ -31,7 +31,7 @@ BaseClient createClient() {
 /// [BaseRequest.followRedirects], and [BaseRequest.maxRedirects] fields. It is
 /// also unable to stream requests or responses; a request will only be sent and
 /// a response will only be returned once all the data is available.
-class BrowserClient extends BaseClient {
+base class BrowserClient extends BaseClient {
```

Comment:
Interface is identical to Client

Reply:
I wonder if this should be `final`?

Reply:
I could imagine people wanting to add methods to `Client` implementations. Like:

````dart
class DownloadingBrowserClient extends BrowserClient {
  Stream<UInt8List> downloadFile(Url) {
     ...
  }

}```